app/uk/gov/hmrc/hmrcemailrenderer/templates/transactionengine/ct/transactionEngineHMRCCTCT600TILSuccess.scala.html

```diff
@@ -1,8 +1,7 @@
 @(params: Map[String,Any])
-@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, s"Successful Receipt of Online Corporation Tax submission for Reference ${params("identifier")}") {
-    <p style="margin: 0 0 30px; font-size: 19px;">Thank you for sending in the online Corporation Tax submission to the 'TEST IN LIVE' service.</p>
+@uk.gov.hmrc.hmrcemailrenderer.templates.helpers.html.template_main(params, s"Successful Online Corporation Tax test submission for Reference ${params("identifier")}") {
+    <p style="margin: 0 0 30px; font-size: 19px;">Thank you for sending in the CT600 submission online to the 'TEST IN LIVE' service.</p>
```

Comment:
You didn't remove "in" here but did in the Failure message above.  Is that intentional?